classes/system/elasticsearch/server/cluster.yml

```diff
@@ -21,19 +21,26 @@ parameters:
   elasticsearch:
     server:
       enabled: true
-      publish_host: ${_param:single_address}
       master: true
       data: false
+      mlockall: true
+      bind:
+        address: ${_param:heka_elasticsearch_host}
```

Comment:
I think `${_param:single_address}` should be used here. `${_param:heka_elasticsearch_host}` is only to be used in the Heka configuration.